django_project/lesson/templates/worksheet/detail.html

```diff
@@ -110,19 +110,21 @@ <h3 class="custom-heading" style="margin-left:-40px"><strong>{{ worksheet.title
     </div>
 
     <div class="row details-worksheet" style="margin-top:10px;">
-        <div class="col-lg-12">
-            <h4><strong>
-                {% trans 'You try:' %}
-            </strong></h4>
-            <p>
-                <strong>
-                    <i>
-                        {% trans 'Goal:' %} {{ worksheet.exercise_goal }}
-                    </i>
-                </strong>
-            </p>
-            <p>{{ worksheet.exercise_task | base_markdown }}</p>
-        </div>
+       {% if worksheet.exercise_goal or worksheet.exercise_task %}
+            <div class="col-lg-12">
+                <h4><strong>
+                    {% trans 'You try:' %}
+                </strong></h4>
+                <p>
+                    <strong>
+                        <i>
+                            {% trans 'Goal:' %} {{ worksheet.exercise_goal }}
+                        </i>
+                    </strong>
+                </p>
+                <p>{{ worksheet.exercise_task | base_markdown }}</p>
```

Comment:
If a worksheet has only `worksheet.exercise_goal`, but no `worksheet.exercise_task`, it will still show empty line also.
Maybe put also if statement here. And also in the goal part.

Reply:
Alright